layout-server/src/main/java/org/finos/vuu/layoutserver/service/LayoutService.java

```diff
@@ -0,0 +1,47 @@
+package org.finos.vuu.layoutserver.service;
+
+import lombok.RequiredArgsConstructor;
+import org.finos.vuu.layoutserver.model.Layout;
+import org.finos.vuu.layoutserver.model.Metadata;
+import org.finos.vuu.layoutserver.repository.LayoutRepository;
+import org.finos.vuu.layoutserver.repository.MetadataRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Service
+public class LayoutService {
+
+    private final LayoutRepository layoutRepository;
+    private final MetadataRepository metadataRepository;
+
+    public Layout getLayout(UUID id) {
+        return layoutRepository.findById(id).orElseThrow();
+    }
+
+    public List<Metadata> getMetadata() {
+        List<Metadata> metadata = new ArrayList<>();
+        layoutRepository.findAll().forEach(layout -> metadata.add(layout.getMetadata()));
+        return metadata;
+    }
+
+    @Transactional
```

Comment:
Let's make sure we test the transactional nature of this.

Reply:
Good point, hadn't necessarily thought about that.

Reply:
Made a note on the test PR https://github.com/ScottLogic/finos-vuu/pull/68#issuecomment-1754889431.